plugin/pkg/scheduler/modeler.go

```diff
@@ -100,6 +103,15 @@ func (s *SimpleModeler) listPods(selector labels.Selector) (pods []api.Pod, err
 	//       need go through that periodically and check for deleted
 	//       pods.
 	for _, pod := range assumed {
+		// Cached objects with expired ttls do not Exist, so this has the
+		// effect of purging the cache and only returning valid objects.
+		podExists, err := s.assumedPods.Exists(&pod)
```

Comment:
I wanted the StoreToPodLister to only return valid pods even if created with a ttlstore, so list does expire them. 
I'll leave the normal eviction to happen based on the other queues and leave the ttl eviction to happen in list.